src/DependencyInjection/Configuration.php

```diff
@@ -140,9 +140,18 @@ public function getConfigTreeBuilder()
                             ->defaultValue($this->debug)
                         ->end()
                         ->scalarNode('formatter')->defaultNull()->end()
-                        ->integerNode('captured_body_length')
+                        ->scalarNode('captured_body_length')
+                            ->beforeNormalization()
+                                ->always(function ($maxLength) {
+                                    if (null === $maxLength) {
+                                        return null;
+                                    }
+
+                                    return (int) $maxLength;
```

Comment:
i'd prefer to throw an InvalidConfigurationOption at this point `if (!is_int($maxLength))`. otherwise one might set it to 0 when accidentally passing a string.